### PR TITLE
Add page down/page up hardware detection

### PR DESCRIPTION
**Pull request by CarlosEsco**
#1202

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 1]
```diff
@@ -206,6 +206,18 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>() {
                         return true
                     }
                 }
+                KeyEvent.KEYCODE_PAGE_DOWN -> {
+                    if (event.action == KeyEvent.ACTION_UP) {
+                        if (!volumeKeysInverted) viewer?.moveDown() else viewer?.moveUp()
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: I think this should be in `onKeyUp` a few lines below. Volume keys had to be handled here to not trigger the system's volume too. Does it make sense to use the inverted volume keys preference too (just asking)?

Comment (CarlosEsco) [reply app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: Will remove and if in future somebody wants add a new invert page property.  Moved to onKeyUp

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt [hunk 2]
```diff
@@ -414,16 +416,16 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>() {
 
     private fun setRotation(rotation: Int) {
         when (rotation) {
-            // Rotation free
+        // Rotation free
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: Can you revert these changes?

Comment (CarlosEsco) [reply app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt]: done

**Comment by CarlosEsco** [comment]
should be good now
